modules/rte_mis_lottery/src/Plugin/DevelGenerate/MiniNodeDevelGenerates.php

```diff
@@ -402,19 +435,20 @@ private function generateParagraph($paragraph_type = '', $data = []) {
         break;
 
       case 'field_school_preferences':
+        // $medium_options = ['hindi', 'english'];
         $values = [
           'type' => 'school_preference',
           'field_school_id' => [
             'target_id' => $data['school_id'] ?? 1,
           ],
           'field_entry_class' => [3],
-          'field_medium' => array_rand(['hindi', 'english']),
+          // 'field_medium' => $medium_options[array_rand($medium_options)],
```

Comment:
Please remove the commented code.

Reply:
This file is only for test purpose. We can ignore this for now.